src/style.css

```
@@ -0,0 +1,77 @@
+@import "./styles/reset.css";
+@import "./styles/stars.css";
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card__image {
+  margin: 32px 19px 0;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin: 40px 16px 0;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 16px 0;
+}
+
+.card__tittle {
+  display: flex;
+  justify-content: space-between;
+  margin: 22px 16px 0;
+}
+
+.card__price {
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+  margin: 0;
+}
+
+.card__costs {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  margin: 0;
+}
+
+.card__buyLink {
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 40px;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin: 16px;
+  color: #fff;
+  background: #00acdc;
```

```suggestion
  background-color: #00acdc;
```
since you use only one special prop from the [background](https://developer.mozilla.org/en-US/docs/Web/CSS/background) better to specify it